bpo-46195: Do not add `Optional` in `get_type_hints` (#30304)
python/cpython: Misc/NEWS.d/next/Library/2021-12-30-21-38-51.bpo-46195.jFKGq_.rst

> Now :func:`typing.get_type_hints` does not add ``Optional`` to types with
> explicit ``None`` default value. This aligns to changes to PEP 484 in

Reviewer: ```suggestion
:func:`typing.get_type_hints` no longer adds ``Optional`` to parameters with
``None`` as a default. This aligns to changes to PEP 484 in
```

> Now :func:`typing.get_type_hints` does not add ``Optional`` to types with
> explicit ``None`` default value.
> 
> This was changed in https://github.com/python/peps/pull/689

Reviewer 2: Minor suggestion to improve clarity:
```suggestion
Now :func:`typing.get_type_hints` does not add ``Optional`` to types with
explicit ``None`` default value. This aligns to changes to PEP 484 in
https://github.com/python/peps/pull/689
```
